sql/parser/walk.go

```diff
@@ -38,14 +38,15 @@ type Visitor interface {
 	VisitPost(expr Expr) (newNode Expr)
 }
 
-func copyQualifiedNames(q QualifiedNames) QualifiedNames {
-	if q == nil {
+// FIXME(knz) Is this really necessary? It seems that unresolved name
```

Comment:
@RaduBerinde I'd like your feedback on this one. I think the entire function can be removed.